<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScorePage</class>
 <widget class="QWidget" name="ScorePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tetris</string>
  </property>
  <widget class="QLabel" name="titleName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Records</string>
   </property>
  </widget>
  <widget class="QLabel" name="place1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1. Alexander (score: 254, line: 16)</string>
   </property>
  </widget>
  <widget class="QLabel" name="place2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2. None</string>
   </property>
  </widget>
  <widget class="QLabel" name="place5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5. None</string>
   </property>
  </widget>
  <widget class="QLabel" name="place3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3. None</string>
   </property>
  </widget>
  <widget class="QLabel" name="place4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4. None</string>
   </property>
  </widget>
  <widget class="QLabel" name="place6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>6. None</string>
   </property>
  </widget>
  <widget class="QLabel" name="place7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>7. None</string>
   </property>
  </widget>
  <widget class="QLabel" name="place8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>8 None</string>
   </property>
  </widget>
  <widget class="QLabel" name="place9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>9. None</string>
   </property>
  </widget>
  <widget class="QLabel" name="place10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>10. None</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
